.github/pull_request_template.md

```diff
@@ -27,31 +27,11 @@ Please explain **IN DETAIL** what the changes are in this PR and why they are ne
 <!-- To manually trigger a benchmark, please check out [Notion](https://www.notion.so/risingwave-labs/Manually-trigger-nexmark-performance-dashboard-test-b784f1eae1cf48889b2645d020b6b7d3). -->
 - [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))
 
-## Documentation
+## Release note
 
-- [ ] My PR contains user-facing changes.
```

Comment:
Does the doc team need to go over all PRs to check release notes? :eyes:

Reply:
> Changes to the code-doc-sync workflow
> - Introduce a script to automatically extract the content in the Release note section for all PRs in a release, to compile the release notes draft.
> - Introduce a GitHub action that automatically creates a doc issue for any PR that has a release note.
> - The doc team reviews the doc issues and determines if this needs to be documented. If not, the issue will be closed.

Oh, I see

Reply:
@hengm3467 What do you think? How about keeping the checkbox here? e.g. `- [ ] My PR needs document changes.`

Reply:
The workflow in my mind is sth like checkbox -> label -> create doc issue. But I think it can be in any form which just works. The doc team can decide it 🤪

Reply:
@fuyufjh Good suggestion! Keeping the checkbox can help people understand the context better.